release-notes.rst

```diff
@@ -1,6 +1,14 @@
 Release Notes
 =============
 
+July 2025
++++++++++
+11 July
+
+- Launchpad no longer supports the creation of new mailing lists. Read more
+  about it `here
```

Comment:
Instead of "here", you could use a more descriptive wording, e.g. "For more information please refer to the announcement."